Replicator/IncomingRev.cc

```diff
@@ -452,12 +452,27 @@ namespace litecore::repl {
         _bodySize       = 0;
     }
 
-    Worker::ActivityLevel IncomingRev::computeActivityLevel() const {
-        if ( Worker::computeActivityLevel() == kC4Busy || _pendingCallbacks > 0 || (_blob != _pendingBlobs.end()) ) {
-            return kC4Busy;
+    Worker::ActivityLevel IncomingRev::computeActivityLevel(std::string* reason) const {
```

Comment:
Is this ever null?  Why not use `std::string&` ?

Reply:
Right now, it's not null, but I am thinking it can serve as a (one-place) switch to turn it off. If it's too verbose, we can simply turn it off and the only overheads are some wasted assignments to local variable k.